thirdeye/thirdeye-pinot/pom.xml

```diff
@@ -92,6 +92,18 @@
       <artifactId>joda-time</artifactId>
     </dependency>
 
+    <!-- dataframe specific -->
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-csv</artifactId>
+      <version>1.3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.udojava</groupId>
```

Comment:
MIT license. It's already a (transitive) part of the ThirdEye dependencies. Alternatively, Exp4j would be an Apache License 2.0 choice.